src/app/swap/[[...path]]/path-resolver.test.ts

```diff
@@ -14,91 +14,109 @@ describe('PathResolver', () => {
     const pathComponents: string[] = []
     const resolver = new PathResolver()
     const resolvedPath = resolver.resolve(pathComponents)
-    expect(resolvedPath.isMinting).toBe(true)
-    expect(resolvedPath.inputToken.symbol).toBe(ETH.symbol)
-    expect(resolvedPath.outputToken.symbol).toBe(indicesTokenList[0].symbol)
+    expect(resolvedPath.isMinting).toBe(false)
+    expect(resolvedPath.inputToken.symbol).toBe(indicesTokenList[0].symbol)
+    expect(resolvedPath.outputToken.symbol).toBe('ETH')
```

Comment:
A lot of these tests had to be changed to make sure that those pairs are now only redeemable.